README.md

```diff
@@ -100,6 +109,102 @@ var client = contentful.createClient({
 
 You can check other options for the client on our reference documentation
 
+## Advanced features
+
+### Link resolution
+
+contentful.js does by default link resolution unless specified otherwise.
+To disable it just set `resolveLinks` to `false` when creating the Contentful client. Like so
+
+```js
+const contentful = require('contentful')
+const client = contentful.createClient({
+  accessToken:'<you-access-token>',
+  space: '<your-space-id>',
+  resolveLinks: false
+})
+```
+
+Please note that the link resolution is only possible when requesting records from the collection endpoint using `client.getEntries()` or by doing  intial sync `client.sync({initial: true})`. In case you want to request one entry and benefit from the link resolution you can use the collection end point with query param `'sys.id': '<your-entry-id>'`.
+
+**e.g.** assuming that you have a contentType `post` that has a reference field `author`
+
+```js
+const contentful = require('contentful')
+const client = contentful.createClient({
+  accessToken:'<you-access-token>',
+  space: '<your-space-id>',
+})
+// getting a specific Post
+client.getEntries({'sys.id': '<entry-id>'}).then((response) => {
+	// output the author name
+	console.log(response.items[0].fields.author.fields.name)
+})
+```
+The link resolution resolves by default one level deep if you have more you can specify the `include` param in the query when fetching your entries like so `client.getEntries({include: <value>})`, you can specify up to 10.
```

Comment:
comma thing again: "resolves, by default". And I'd change "have" to "need"
Then break it into two sentences as right now it's a run-on sentence:
```
The link resolution resolves, by default, one level deep. If you need more
```